jsonapi_requests/request_factory.py

```diff
@@ -31,7 +31,7 @@ def request(self, api_path, method, *, object: data.JsonApiObject = None, **kwar
         if object is not None:
             assert 'json' not in kwargs
             kwargs['json'] = {'data': object.as_data()}
-        return self.retrying.call(self._request, url, method, **kwargs)
+        return self.retrying.__call__(self._request, url, method, **kwargs)
```

Comment:
the Pythonic way to do this is`return self.retrying(self._request, url, method, **kwargs)` in my opinion. That does seem to work as expected.

Reply:
thanks for fixing my non Pythonic ways